### PR TITLE
Blogpost:Adding configuration on how to debug Apache Camel with the Vimspector

### PR DESCRIPTION
**Pull request by Navyashree0923**

### CODE REVIEW

File: content/blog/2022/09/vimdap/index.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
+---
+title: "Configure Camel Debug Adapter in Vim to enable Camel Textual Route debugging"
```
Comment (apupier) [inline content/blog/2022/09/vimdap/index.md:2]: as a title, I'm wondering if we cannot make it shorter and emphasize the important part fo rCamel users. Something like: "Camel Textual Route debugging in Vim"
Your title is fine too, just tryign to give more ideas

File: content/blog/2022/09/vimdap/index.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+}
+```
+* Compile the source to reflect the new changes
+ ```sh
+ mvn compile
+ ```
+* Launch this example with the profile `camel.debug`
+```sh
+ mvn camel:run -Pcamel.debug
+```
```
Comment (apupier) [inline content/blog/2022/09/vimdap/index.md:155]:
```suggestion
* Launch this example in Camel debug mode:
```sh
 mvn camel:debug
```
```

File: content/blog/2022/09/vimdap/index.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+## Attach this camel example to Vimspector for Camel Textual Route Debugging in window-2
+* Open this file in vim `src/main/java/org/apache/camel/example/MyRouteBuilder.java` and then use the following keys to set the breakpoints.
+* Start the Camel debugger by again using the following keys.
+* Enter `1099` as port number when it is prompted for user input in vimspector.
+* Now you could able to debug your camel application and again use the following key mappings to instruct the debugger.
```
Comment (apupier) [inline content/blog/2022/09/vimdap/index.md:160]:
```suggestion
* Now you should be able to debug your camel application and again use the following key mappings to instruct the debugger.
```

**Comment by Navyashree0923** [comment]
Hello @apupier, I have add the changes, please review it.

File: content/blog/2022/09/vimdap/index.md [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+
+# VIM Debug Adapter client for Apache Camel
+
+I just got the debug integration for camel in vim running using [vimspector](https://github.com/puremourning/vimspector).
+
+![Demo](images/vimdap.gif)
+
+## Install
+Install the vimspector using vim-plug plugin manager.
+- Inside `.vimrc` file:
```
Comment (apupier) [inline content/blog/2022/09/vimdap/index.md:19]: the rendering seems bad on the website. I think you need to add an empty line

image

Comment (Navyashree0923) [reply content/blog/2022/09/vimdap/index.md]: Thanks. I have updated the change

**Review by apupier — APPROVED** [review]

**Pull request merged**